tomodachi/transport/aws_sns_sqs.py

```diff
@@ -254,6 +284,7 @@ async def subscribe_handler(
         message_envelope: Any = MESSAGE_ENVELOPE_DEFAULT,
         message_protocol: Any = MESSAGE_ENVELOPE_DEFAULT,  # deprecated
         filter_policy: Optional[Union[str, FilterPolicyDictType]] = FILTER_POLICY_DEFAULT,
+        topic_type: TopicType = TopicType.STANDARD,
```

Comment:
Not happy about this name either really since this setting applies both to the queue and the topic. Let me know if you have any other naming suggestion.